src/main/java/chatBot/bot/Parser.java

```diff
@@ -0,0 +1,125 @@
+package chatBot.bot;
+
+import chatBot.command.*;
+import chatBot.exception.EmptyDescException;
+import chatBot.exception.InvalidCommandException;
+import chatBot.task.Deadline;
+import chatBot.task.Event;
+import chatBot.task.Task;
+import chatBot.task.ToDoTask;
+
+import java.time.format.DateTimeParseException;
+
+/**
+ * Parser is a class to parse input Strings. It has 2 methods
+ *
+ * parseTask method is used exclusively for parsing tasks from a storage file,
+ * returns a Task to the caller.
+ *
+ * parse method is used to determine the command to go to next.
+ * returns a Command to the caller.
+ */
+
+public class Parser {
+    static Task parseTask(String s) {
```

Comment:
Should this String and the Task variable later be named more descriptively, as you are using them throughout the method?